### PR TITLE
Add testcase with IsIPv4Localhost in  resolvconf_test.go

### PR DESCRIPTION
**Pull request by chchliang**
Signed-off-by: chchliang <chen.chuanliang@zte.com.cn>

Add testcase with IsIPv4Localhost in  resolvconf_test.go

### CODE REVIEW

**Comment by chchliang** [comment]
@vdemeester 
I added this PR，but `ci/circleci` show `Your tests failed on CircleCI`, i clicked `Details` and no found fail reason.

**Comment by fcrisciani** [comment]
@chchliang the reason of the last iteration was:
```
*** Test killed with quit: ran too long (10m0s).
FAIL	github.com/docker/libnetwork/networkdb	600.006s
```
But looks like one of the test remained stuck, I retriggered it

**Comment by chchliang** [comment]
@fcrisciani 
`ci/circleci — Your tests failed on CircleCI ` rebuild fail again.

File: resolvconf/dns/resolvconf_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package dns
+
+import (
+	"strconv"
+	"testing"
+)
+
```
Comment (fcrisciani) [inline resolvconf/dns/resolvconf_test.go]: try to add this: `var _ = flag.Bool("incontainer", false, "Indicates if the test is running in a container")`

**Comment by fcrisciani** [comment]
ping @chchliang

File: resolvconf/dns/resolvconf_test.go (hunk 1 above)
Comment (chchliang) [reply resolvconf/dns/resolvconf_test.go]: Why does add this? I don't understand.

**Pull request closed without merging**